HealthcareApp/src/Healthcare.Infrastructure/DependencyInjection.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Immutable;
+using System.Text;
+using Application.Abstractions;
+using Application.Abstractions.Decorators;
+using Domain.Entities;
+using Healthcare.Infrastructure.Persistance;
+using Infrastructure.Configuration;
+using Infrastructure.Repository;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Infrastructure;
+
+public static class DependencyInjection
+{
+    public static IServiceCollection AddInfrastructure(this IServiceCollection services, IConfiguration configuration)
+    {
+        var connection = configuration.GetConnectionString("HealthcareDb");
+
+        services.AddDbContext<AppDbContext>(
+            options => options.UseSqlServer(connection)
+        );
+        services.AddIdentity<ApplicationUser, IdentityRole>(options =>
+            {
+                options.User.AllowedUserNameCharacters = "";
+                options.Password.RequireNonAlphanumeric = false;
+            })
+            .AddEntityFrameworkStores<AppDbContext>()
+            .AddDefaultTokenProviders();
+
+        services.AddAuthentication(options =>
+        {
+            options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+            options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+        }).AddJwtBearer(options =>
+        {
+            options.TokenValidationParameters = new TokenValidationParameters
+            {
+                ValidateIssuer = true,
+                ValidateAudience = true,
+                ValidateIssuerSigningKey = true,
+                ValidIssuer = configuration["Jwt:Issuer"],
+                ValidAudience = configuration["Jwt:Audience"],
+                IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(configuration["Jwt:Key"] ?? ""))
+            };
+        });
+
+        services.AddScoped<IUserAuthenticationService, AuthenticationService>();
```

Comment:
AuthenticationService is business logic. It's not logical to register it with infrastructure services